EmployeeManagementService/EmployeeManagementService.Domain/Services/EmployeeService.cs

```diff
@@ -14,11 +14,13 @@ public interface IEmployeeService
         Task CreateEmployee(Employee newEmployee, string password);
         Task EmployeeLogIn(string username, string password);
         Task EmployeeLogout(long id);
+        Task<Employee> GetAddressByEmployeeId(long id);

```

Comment:
We don't really need a GetAddress or UpdateAddress.

This should just be done when we UpdateEmployee or CreateEmployee or GetEmployee